docs/i18n.md

```diff
@@ -9,96 +9,99 @@ but for apps as well.
-	
-	*Note:* This is just asset registration, no data will be fetched at that point.
-	
-4. Fetch the texts for the current language:
-
-	```js
-	await fetchI18nBundle("myApp");
-	```
-	
-	This will fetch the appropriate texts for the currently configured language.
-	
-5. Get and use the bundle
-
-	```js
-	const bundle = getI18nBundle("myApp");
-	const pleaseWait = bundle.getText("PLEASE_WAIT");
-	console.log("Please wait in the current language is: ", pleaseWait);
-	```
-	
-	You can pass multiple additional values to `getText` for texts with placeholders, for example:
-	
-	If your text looks like this:
-	
-	`CAROUSEL_DOT_TEXT=Item {0} of {1} displayed`
-	
-	you can call `getText` like this:
-	
-	`bundle.getText("CAROUSEL_DOT_TEXT", 5, 20);`
-	
-	which will finally result in, for example:
-	
-	`Item 5 of 20 displayed`. 
-
-6. Test your page using different languages, f.e. set `?sap-ui-language=de` in the URL or change the configuration.	
-		
+### 1. Start by creating some `i18n` resources in a directory that can be served, for example:
+
+|                 File                 |          Content           |
+| ------------------------------------ | -------------------------- |
+| `assets/messagebundle_de.properties` | `PLEASE_WAIT=Bitte warten` |
+| `assets/messagebundle_fr.properties` | `PLEASE_WAIT=Patientez.`   |
+| `assets/messagebundle_es.properties` | `PLEASE_WAIT=Espere`       |
+| `assets/messagebundle_en.properties` | `PLEASE_WAIT=Please wait`  |
+
+
+### 2. Import the following `i18n`-related modules to your app:
+
+```js
+import parse from "@ui5/webcomponents-base/dist/PropertiesFileFormat.js";
+import { registerI18nLoader, fetchI18nBundle, getI18nBundle  } from "@ui5/webcomponents-base/dist/asset-registries/i18n.js";
+```
+
+The first one provides support for `.properties` files, as used in the example and the second one - the functions
+that will allow you to take advantage of the `i18n` functionality.
+
+### 3. Register your a loader function that can retrieve and process the actual content of your message bundles:
+
+```js
+const supportedLocales = ["en", "fr", "de", "es"];
+supportedLocales.forEach(localeToRegister => {
+	registerI18nLoader("myApp", localeToRegister, async (localeId) => {
+		const props = await (await fetch(`./assets/messagebundle_${localeId}.properties`)).text();
+		return parse(props);
+	});
+};
+```
+
+The first argument to `resisterI18nLoader` is an ID that will be used to reference this message bundle, the second is the locale this loader can load and the third is is a function that can load (and process) the content for the specified package/locale combination.
+
+*Note:* For more assets, a loop is used to register a loader for each package/locale combination. The same loader function can be registered and its parameter can be used to distinguish which locale resource is requested.
+
+*Note:* This is just asset registration, no data will be fetched at that point.
```

Comment:
This step takes care of registering assets only, no data will be fetched at this point.